### ISSUE TITLE
attempt fix failed too many times

### ISSUE THREAD

**Issue by mhspadez** [stackblitz/bolt.new#3801]
### Describe the bug

I'm trying to create my first page. I haven't been able to get a working version even though I bought the first tier. I'm positive it can create what I'm looking for so I'm excited to use it, but I keep getting a failed attempt to fix. I found this thread on github which basically says this is a known issue but the solution is to just keep hitting "attempt fix" a ton of times and it will eventually work -- which is my exact issue. (thread here: https://github.com/stackblitz/bolt.new/issues/176) The problem is I ran out of credits while attempting this fix. I'm sure this isn't what the limitation on the credits is supposed to be used for (burn credits on 1 single fix?), I don't even have a single page to test even once. Would it be out of the question to restore some credits just to get this bug working? Thank you!

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-swrles

### Steps to reproduce

I only have 1 project, just hit "attempt fix"
![Capture](https://github.com/user-attachments/assets/1fc7b0c6-d95c-4624-a634-16bce7a06415)

### Expected behavior

the AI was attempting to fix an issue, I even asked it for several ways to get around it, but it doesn't seem to be able to do it. Expected behavior is to just fix it or give me some solution to fix it -- provide some kind of info, suggestions on moving it to another context, not sure. I'm not really a coder, but it seems like it should be a simple install right?

### Screen Recording / Screenshot

This doesn't seem necessary, but let me know if you need more screenshots

### Platform

```
Browser name = Chrome
Full version = 131.0.0.0
Major version = 131
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 83254426,
  "usedJSHeapSize": 76278458,
  "jsHeapSizeLimit": 4294705152
}
Username = mhfx
Chat ID = 8a81cf200a52
```

**Comment by mhspadez** [comment]
Is there a way we can run the install manually some how? Or upload the dependency files?

**Comment by mhspadez** [comment]
I was able to solve this myself. The npm install failure shows the repository it was trying to install from and giving a 404 not found error. It was because it was trying to install using an outdated npm install code that led to an empty page. In my case it was something like npm install assemblyai/@core (can't remember the exact install command it was using), when the command should have just been "npm install assemblyai". Ran this myself in the terminal and that fixed this issue.
